lib/grammar2.pegjs

```diff
@@ -72,7 +71,13 @@ ctrl = ifctrl
      / forctrl
      / returnctrl
 
-ifctrl     = "if" _ c:innerstmt _ b:block { return p.parseIf(c, b) }
+ifctrl = "if" _ c:innerstmt __ b:block ei:(__ elseifcont)* {
```

Comment:
Yup! PEG.js uses the the PCRE convention of asterisk for zero-or-more, plus for one-or-more, and question mark for zero or one.